src/Initializer/MemoryManager.h

```diff
@@ -362,6 +362,10 @@ class seissol::initializers::MemoryManager {
       return getUnsafe<std::string>((*m_inputParams)["output"], "outputfile");
     }
 
+    bool isLoopStatisticsNetcdfOutputOn() const {
```

Comment:
Just to put it out here: we're slowly building a parameter reader inside the `MemoryManager` class, and I am not sure if it is a good place to loosely put it in there like that. However, discussing/changing this is probably off-topic for this PR. I'd say it's good like that for now.

(still, just as a side note: there is some rather complete parameter reader on the branch `davschneller/no-fortran`, right now in `Initializer/SeisSolParameters.hpp`; but the branch is not ready to merge so far unfortunately ...)